kcidb/__init__.py

```diff
@@ -14,6 +14,9 @@
 # Compatibility alias
 io_schema = io.schema
 
+# latest kcidb-io version
+io_latest_version = io_schema.V4
```

Comment:
Now that we've merged the base work both in kcidb-io and kcidb, please rebase your PR and use `kcidb.io.SCHEMA` to refer to the currently-used schema instead.